samples/ai-platform/Chicago Crime Pipeline.ipynb

```diff
@@ -31,7 +31,7 @@
     "%%capture\n",
     "\n",
     "# Install the SDK (Uncomment the code if the SDK is not installed before)\n",
-    "KFP_PACKAGE = 'https://storage.googleapis.com/ml-pipeline/release/0.1.22/kfp.tar.gz'\n",
+    "KFP_PACKAGE = 'kfp'\n",
```

Comment:
How about we just remove this variable altogether, and have the line read everywhere as:
`!pip3 install kfp --upgrade`

Reply:
It'll also be more consistent across notebooks.

Reply:
+1. Its becoming cumbersome maintaining like this

Reply:
SG.